NAS-100336 Add docs for cloud sync stop button (#1177)
freenas/freenas-docs: userguide/tasks.rst

:menuselection:`Tasks --> Cloud Sync Tasks` is used to define the
schedule for running a cloud sync task. The time selected is when
the Cloud Sync task is allowed to begin. The cloud sync runs until
finished, even after the time selected. To stop the cloud sync task

> before it is finished click

Reviewer: ```suggestion
before it is finished, click
```

   Cloud Sync Status

When a cloud sync task has run, :literal:`SUCCESS`,
:literal:`FAILURE`, or :literal:`ABORTED` is shown. :literal:`ABORTED`

> is shown if a cloud sync is stopped while it was running. If a cloud

Reviewer: ```suggestion
is shown when a cloud sync was stopped before completion. If a cloud
```

> sync task has failed, a short description of why it failed is displyed

Reviewer: ```suggestion
sync task failed, a short description of why it failed is displayed
```

> after :literal:`FAILURE`. Click the status of the cloud sync task to

Reviewer: Not clear which button or element is the status. Also, the next three sentences are kind of unclear.
"Click (something) to see the task logs. Click :guilabel:`DOWNLOAD LOGS` to download the log files."
